qutebrowser/browser/commands.py

```diff
@@ -1175,6 +1175,25 @@ def tab_move(self, index: typing.Union[str, int] = None, count=None):
         cmdutils.check_overflow(new_idx, 'int')
         self._tabbed_browser.tabBar().moveTab(cur_idx, new_idx)
 
+    @cmdutils.register(instance='command-dispatcher', scope='window')
```

Comment:
Should probably have `hidden=True` as you woudln't call it manually

Reply:
I don't get where to add it, adding it to
```python
@cmdutils.register(instance='command-dispatcher', scope='window')
```
leads to a crash on my device.

Reply:
Sorry, `hide=True`

Reply:
Also leads to a crash.
Decorator saves kwargs: https://github.com/qutebrowser/qutebrowser/blob/master/qutebrowser/commands/cmdutils.py#L77
Command creation: https://github.com/qutebrowser/qutebrowser/blob/master/qutebrowser/commands/cmdutils.py#L111
Command constructor doesn't allow to assign an argument to a parameter named hide/hidden: https://github.com/qutebrowser/qutebrowser/blob/b31360b6e3cdd1bda27f63c6dcc42934ce27db70/qutebrowser/commands/command.py#L78
The only way to hide a command seems to be debug=True (or to exclude it to be used in a mode).

Reply:
Argh, you're right. We just hide commands that aren't valid in normal mode. Ah well, not a big deal.

Reply:
There used to be a `hide=True`, but it confused a lot of people, so I removed it.